.github/workflows/ci-cmake.yaml

```diff
@@ -36,13 +36,15 @@ jobs:
 
   build-win:
     name: build-win
-    runs-on: windows-2019
+    runs-on: windows-2022
     needs: draft-release
     permissions:
       contents: write
     env:
       PreferredToolArchitecture: x64
       TIPI_DISTRO_MODE: all
+      TIPI_DISTRO_JSON: https://raw.githubusercontent.com/tipi-build/distro/2ce62a7fd963d2bf89cc50184c859e2198f9ef64/distro.json
+      TIPI_DISTRO_JSON_SHA1: 75756944827cbfde68d6dbc7b3f77584420cdc9c
```

Comment:
This conflicts with the distro.json stored within .tipi/ and imported via .tipi/env. I don't think we want these kind of differences, we should update the locally used distro or remove it, but then I think the linux build will be incorrect (needed shared objects from the stdlib)

Reply:
I think I can remove the local distro store.
the environment commit on this distro I'm now based on (just some commit more after now )